src/ImageRunModal.jsx

```diff
@@ -384,6 +401,15 @@ export class ImageRunModal extends React.Component {
                 })));
         }
 
+        // Query if used url is a valid container that can be pulled
+        if ((value.includes('.') || value.startsWith("localhost")) && value.includes('/')) {
```

Comment:
oh yeah this was only for dev testing, needs to be improved

Reply:
I ignore this for now. What's that supposed to do? But the "localhost" special case must go, otherwise the integration test is meaningless.

Reply:
It's extremely ugly but this is meant to only fire up the manifests query when user input looks like container image url. It could be just `value.includes('/')`  but I feel like I should come up with something better

Reply:
with "image URL" you mean something like "ghcr.io/owner/name:tag"? (that's not an URL). I'm not sure if that can be captured with a regex -- registries can be pretty much arbitrary and don't *have* to contain a `.` (e.g. `localhost` or a custom defined one), and "localhost" by itself isn't special. Frankly, your update to just check `/` seems to be the best that you can do -- try to query it if it's a valid image/manifest, and if not then use the term as a search query. Or do both in parallel?